packages/map-popup/src/index.tsx

```diff
@@ -63,17 +63,18 @@ const StationHubDetails = ({ station }: { station: Station }) => {
 const StopDetails = ({ id, setViewedStop }: { id: string, setViewedStop: () => void; }) => {
   return (
     <Styled.PopupRow>
-      <strong>
-        <FormattedMessage
-          defaultMessage={defaultMessages["otpUi.MapPopup.stopId"]}
-          description="Displays the stop id"
-          id="otpUi.MapPopup.stopId"
-          values={{
-            stopId: id
-          }}
-        />
-      </strong>
-      <ViewStopButton onClick={setViewedStop}>
+      { id &&
```

Comment:
Can we remove this extra space?

```suggestion
      {id &&
```